tools/serverpod_cli/lib/src/util/exit_exception.dart

```diff
@@ -0,0 +1,24 @@
+enum ExitCodeType {
+  /// General errors - This code is often used to indicate generic or
+  /// unspecified errors.
+  general(1),
```

Comment:
We are missing an exit code here.

`invalidArgument` 128 (this should also be used over `commandInvokedCannotExecute` in some places above.

Reply:
Status code 128 has a bit of a different meaning, it has more to do with having wrong arguments for the process exit method itself than the state of the program: https://tldp.org/LDP/abs/html/exitcodes.html.

Will keep as is.